apps/website/src/server/db/show-and-tell.ts

```diff
@@ -91,6 +91,7 @@ const showAndTellSharedInputSchema = z.object({
   text: z.string().max(1_000),
   imageAttachments: imageAttachmentsSchema,
   videoLinks: videoLinksSchema.max(MAX_VIDEOS),
+  volunteeringMinutes: z.number().int().nonnegative().nullable(),
```

Comment:
Should this be positive? Allowing someone to submit 0 hours seems silly -- they should just not check the box?